portfolio/src/main/webapp/script.js

```diff
@@ -145,4 +145,93 @@ function dark() {
     document.getElementById('poem_1').style.backgroundColor = 'black';
     document.getElementById('poem_2').style.backgroundColor = 'black';
     document.getElementById('poem_3').style.backgroundColor = 'black';
-}
\ No newline at end of file
+}
+
+var amplitude = 70;
+var left_pos = 600;
+var top_pos = -600;
+var pi = Math.PI;
+var inc = pi / 30;
+var params = new Map();
+params.set("one", 0).set("two", 0).set("three", 0).set("four", 0).set("five", 0);
```

Comment:
This looks like a good place to use a for loop instead of repeating the same statement ten times.